docs/examples/gen_velocities.rst

```diff
@@ -0,0 +1,432 @@
+is perpedicular to the lens and latitude parallel to the lens. When converting
+positions or velocities in this frame to cartesian representation, the x-axis
+will point along the line of sight, the y-axis perpendicular to the screen, and
+the z-axis parallel to the screen (in the direction of its motion). Hence, we
+need to compute the cartesian z-component of velocities in ``lens_frame``.
+
+Calculating effective velocities
+================================
+
+There are several components of the effective velocity that can be computed
+separately:
+
+.. list-table::
+    :widths: 2 1
+    :header-rows: 1
+
+    * - Velocity component
+      - Symbol
+    * - Earth's velocity as a function of time
+      - :math:`v_\mathrm{E}( t )`
+    * - pulsar's orbital velocity as a function of time
+      - :math:`v_\mathrm{p,orb}( t )`
+    * - pulsar system's barycentric velocity
+        (corresponding to the proper motion)
+      - :math:`v_\mathrm{p,pm}`
+    * - velocity of the interstellar medium at the lens (known in this example)
+      - :math:`v_\mathrm{ISM}`
+
+All these refer to the component of the velocity projected onto the
+one-dimensional lens.
+
+Earth's velocity
+----------------
+
+To obtain Earth's velocity in the lens frame, first generate a location on
+Earth's surface using the :py:class:`~astropy.coordinates.EarthLocation` class
+(in this case the location of the Parkes radio telescope). This class has the
+:py:meth:`~astropy.coordinates.EarthLocation.get_gcrs` method, which returns
+positions (with respect to the centre of the Earth) as a function of time.
+These are transformed into the lens frame using the
+:py:meth:`~astropy.coordinates.BaseCoordinateFrame.transform_to` method.
+Velocities can then be extracted using the
+:py:attr:`~astropy.coordinates.BaseCoordinateFrame.velocity` attribute, and
+finally :py:attr:`~astropy.coordinates.CartesianDifferential.d_z` isolates the
+z-component of the velocity (in the direction of the screen).
+
+.. plot::
+    :include-source:
+    :context:
+
+    earth_loc = EarthLocation('148°15′47″E', '32°59′52″S')
+    
+    v_earth = earth_loc.get_gcrs(t).transform_to(lens_frame).velocity.d_z
+
+Pulsar's orbital velocity
+-------------------------
+
+Compute the pulsar's orbital velocity projected onto the screen
+    
+.. math::
+
+    v_\mathrm{p,orb} = - \frac{ K_\mathrm{p} }{ \sin( i_\mathrm{p} ) }
+                         \left[ \cos( i_\mathrm{p} )
+                                \sin( \mathit{\Delta\Omega}_\mathrm{p} )
+                                \cos( \theta_\mathrm{p} )
+                              - \cos( \mathit{\Delta\Omega}_\mathrm{p} )
+                                \sin( \theta_\mathrm{p} )
+                         \right].
+
+Here, :math:`\theta_\mathrm{p}( t )` is the phase of pulsar orbit as measured
+from its ascending node.
+
+.. plot::
+    :include-source:
+    :context:
+
+    th_p = ((t - t0_p) / p_b).to('') * u.cycle
```

Comment:
I prefer `.tp(u.one)` or `.to(1)`

Reply:
I changed it to `u.dimensionless_unscaled`, which I wanted to avoid because it's such a mouthful, but I found `.to(u.one)` and `.to(1)` also a bit confusing, and the [astropy documentation](https://docs.astropy.org/en/stable/units/standard_units.html#doc-dimensionless-unit) states that
> For compatibility with the supported unit string formats, this is equivalent to Unit('') and Unit(1), though using u.dimensionless_unscaled in Python code is preferred for readability